src/audio.c

```diff
@@ -455,5 +455,5 @@ static void addAudio(Audio * root, Audio * new)
         root = root->next;
     }
 
-    root->next = new;
+    root->next = newAudio;
```

Comment:
Last change, if you'd like you can add yourself to the bottom of the README as a contributor:

```
* [Firstname Lastname](https://github.com/Karlovsky120)
```

Otherwise, I'll merge

Reply:
Sure, why not. I'll squash that into the last commit.